cmd/appsctl/aws.go

```diff
@@ -235,9 +235,7 @@ var awsTestLambdaCmd = &cobra.Command{
 		}
 
 		creq := apps.CallRequest{
-			Call: apps.Call{
-				Path: "/ping",
-			},
+			Call: *apps.NewCall("/ping"),
```

Comment:
Does a builder pattern as described [here](https://devcharmander.medium.com/design-patterns-in-golang-the-builder-dac468a71194) make more sense for constructing calls? That way we should get rid of the de-referencing.

Reply:
Agree, but perhaps a separate (HW?) ticket?

Reply:
Sound good to me. Would you please file one?

Reply:
Filed https://mattermost.atlassian.net/browse/MM-40508